app/models/form.rb

```diff
@@ -157,16 +157,25 @@ def depends_on_met(depends_on, case_log)
     depends_on.any? do |conditions_set|
       return false unless conditions_set
 
-      conditions_set.all? do |question, value|
+      conditions_set.all? do |condition|
+        object, method, value = condition.values_at("object", "method", "value")
+        object_instance = case object
+                          when "case_log"
+                            case_log
+                          when "user"
+                            nil
+                          end
```

Comment:
Could maybe shorten it to use a ternary operator I think? 
`object_instance = object == "case_log" ? case_log : nil`

Reply:
Sort of deliberate doing this to indicate where the next PR is going. i.e. "user" is an expected value that doesn't do anything yet but will.